Rakefile

```diff
@@ -0,0 +1,8 @@
+begin
+  require 'bundler/setup'
+rescue LoadError
+  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
```

Comment:
Do you want to re-raise the error here? Or exit? Also, maybe substitute `puts` with `warn`.